gatsby/plonerestapi.rst

```diff
@@ -0,0 +1,131 @@
+Fetching Data Using Plone REST.API
+==================================
+
+All the data from a Plone site is available in the JSON format using the `plone.restapi <https://plonerestapi.readthedocs.io/en/latest/introduction.html>`_.
```

Comment:
Maybe you should mention the purpose of the plugin that we are building from scratch, before talking about plone.restapi.
Maybe the better place is in the previous chapter, so users knows that we're getting contents from a Plone site, and here you explain how restapi works